sequencing/vcf_to_plink/Readme.md

```diff
@@ -1,15 +1,13 @@
-# vcf_to_plink (DNAnexus Platform App)
+# Convert VCF file to PLINK file (DNAnexus Platform App)
 
-Uses PLINK 1.9 to convert VCF files to PLINK bed/bim/fam files.
+## What does this applet do?
 
-## What does this app do?
+This app uses PLINK 1.9 to convert VCF files (`*.vcf.gz`) to PLINK bed/bim/fam files (`*.bed`, `*.bed`, `*.fam`).
 
 ## What data are required for this app to run?
 
-## Usage
-### Options
+This app requires an equal number of VCF files (`*.vcf.gz`) and their associated index files (`*.vcf.gz.tbi`).
 
-## What does this app output?
+## What does this applet output?
```

Comment:
What does this applet output?" -> "What does this app output?"
I am assuming that all applets will eventually be published as apps, for the paper's publication